src/vs/base/common/types.ts

```diff
@@ -42,6 +42,15 @@ export function isObject(obj: unknown): obj is Object {
 		&& !(obj instanceof Date);
 }
 
+/**
+ *
+ * @returns whether the provided parameter is of type `Buffer` or Uint8Array dervived type
+ */
+export function isBuffer(obj: unknown): obj is Object {
+	return typeof obj === 'object'
+		&& obj instanceof Uint8Array;
```

Comment:
I assume that all the types in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays#typed_array_views will have the same problem, I think this should check for those too. Should be called `isTypedArray` or something like that

Reply:
Yes seems obvious in retrospect. I'll change it.